backend/open_webui/migrations/versions/64d8cd268d63_add_allowed_domains_to_groups_table.py

```diff
@@ -0,0 +1,29 @@
+"""add allowed_domains to groups table
+
+Revision ID: 64d8cd268d63
+Revises: a71ba3c6d0a4
+Create Date: 2025-09-02 21:45:34.156835
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "64d8cd268d63"
+down_revision: Union[str, None] = "a71ba3c6d0a4"
```

Comment:
We're gonna need to be careful when merging here due to other PRs:
https://github.com/ssc-dsai/canchat-v2/pull/232/files#diff-913fbda72496fd26dc46acee231a845f05e62f92f7e883a3285c11a4a066264bR18